lighthouse-core/gather/gatherers/dobetterweb/domstats.js

```diff
@@ -0,0 +1,132 @@
+/**
+ * @license
+ * Copyright 2017 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * @fileoverview Gathers stats about the max height and width of the DOM tree
+ * and total number of nodes used on the page.
+ */
+
+/* global document */
+
+'use strict';
+
+const Gatherer = require('../gatherer');
+
+/**
+ * Constructs a pretty label from element's selectors. For example, given
+ * <div id="myid" class="myclass">, returns 'div#myid.myclass'.
+ * @param {!HTMLElement} element
+ * @return {!string}
+ */
+/* istanbul ignore next */
+function createSelectorsLabel(element) {
```

Comment:
It does. Here's what it looks like:

<img width="417" alt="screen shot 2017-02-16 at 6 30 06 pm" src="https://cloud.githubusercontent.com/assets/238208/23050261/092967a4-f476-11e6-9d5e-1336ca5e5718.png">